src/helpers/decodeTxData.ts

```diff
@@ -29,7 +30,7 @@ module.exports = function(functionSignature, txData) {
   const paramsStr = paramsMatch[1]
   const params = paramsStr.split(',')
 
-  const args = abi.decodeParameters(params, txData.slice(10))
+  const args = abi.decodeParameters(params, '0x' + txData.slice(10))
```

Comment:
Q: What do you think about to use the function `add0x` in this line @fvictorio ?

Reply:
Good catch, fixed.